test/distributed/test_c10d_common.py

```diff
@@ -307,6 +313,360 @@ def _prepare_multi_device_module(
 
         return model, ddp_model, input, target
 
+    def _get_store(self):
+        return dist.FileStore(self.file_name, self.world_size)
+
+    def _get_process_group(self):
+        raise ValueError("To be implemented by child class")
```

Comment:
nit: I think traditionally we raise `NotImplementedError()` if we are not using abstract base classes.